uui/components/tables/DataTableHeaderCell.module.scss

```diff
@@ -20,7 +20,7 @@
     padding-inline-end: var(--uui-dt-header-cell-padding-end);
     width: 0;
     background-clip: padding-box;
-    height: var(--uui-dt-header-cell-height);
+    min-height: var(--uui-dt-header-cell-height);
```

Comment:
It is required for DataTableHeaderCell to expand to 100% height if groups are enabled.